### DISCUSSION TITLE
共享文件夹是用来做什么的？为什么以前允许自己设置，f3ad724 改到内部目录了？

### DISCUSSION THREAD

**Discussion by Henry-ZHR** [osfans/android-rime#1465] [Q&A | 问答] [answered]
防止被说是 [X-Y 问题](https://en.wikipedia.org/wiki/XY_problem)，以下是我目前的情况：

我之前用共享文件夹存储从电脑端同步过来的方案的文件，这些文件是通过包管理器安装到系统且经常更新的，使用 Syncthing 同步；修改之后因为 documentsui 看不见 Android/data 目录所以 Syncthing 没法把文件直接同步到共享文件夹  
用 Syncthing 把文件直接同步到用户文件夹应该是可行的，但这样一来自动同步的方案文件和手动编辑的配置文件都放在一起，不方便管理

**Comment by Henry-ZHR** [comment]
@WhiredPlanck

**Comment by WhiredPlanck** [comment, marked as answer]
@Henry-ZHR 共享文件夹用于放置内置（自带）或者通用的数据，例如内置的“默认”和“同文风”主题、Rime 的基础文件（`default.yaml` 等）和 OpenCC 的默认转换配置和字典文件等。这个文件夹的内容不鼓励用户直接更改，因为它们会随软件的更新而更新。

要了解更多，请参见 Rime 维基中的[用户文件夹](https://github.com/rime/home/wiki/UserData)和[共享文件夹](https://github.com/rime/home/wiki/SharedData)章节。
